src/block.js

```diff
@@ -38,108 +40,233 @@ module.exports = (common) => {
 
     after((done) => common.teardown(done))
 
-    describe('.put', () => {
-      it('a buffer, using defaults', (done) => {
-        const expectedHash = 'QmPv52ekjS75L4JmHpXVeuJ5uX2ecSfSZo88NSyxwA3rAQ'
-        const blob = new Buffer('blorb')
+    describe('callback API', () => {
```

Comment:
@haadcode I believe you missed that there is no more separation between Callback API and Promise API and through your rebase you brought it back and duplicated some come.

Reply:
I did not notice that. Will fix it.

Reply:
@diasdavid I started to look into this but I'm little confused. Can you help here to elaborate what you mean with "no more separation between Callback and Promise API"? We still have them both, right?

Reply:
We still have both, but we don't duplicate the tests anymore :)